nvflare/cli.py

```diff
@@ -125,6 +127,19 @@ def handle_config_cmd(args):
     print_hidden_config(config_file_path, nvflare_config)
 
 
+def def_pre_install_parser(sub_cmd):
+    cmd = CMD_PRE_INSTALL
+    try:
+        # using try catch to avoid hard dependency on nvflare.tool.code_pre_installer
+        from nvflare.tool.code_pre_installer.install import define_pre_install_parser
```

Comment:
why do we need the try-catch for this one?
all our other commands are just directly import